e2e-polybft/e2e/bridge_test.go

```diff
@@ -837,10 +837,11 @@ func TestE2E_Bridge_ChildchainTokensTransfer(t *testing.T) {
 
 			// check that balances on the child chain are correct
 			for i, receiver := range depositors {
+				temp := big.NewInt(0)
 				balance := erc20BalanceOf(t, receiver, contracts.NativeERC20TokenContract, childchainTxRelayer)
 				t.Log("Attempt", it+1, "Balance before", balancesBefore[i], "Balance after", balance)
 
-				if balance.Cmp(balancesBefore[i].Add(balancesBefore[i], big.NewInt(amount))) != 0 {
+				if balance.Cmp(temp.Add(balancesBefore[i], big.NewInt(amount))) != 0 {
```

Comment:
We can get rid off the temp var.

```suggestion
				if balance.Cmp(new(big.Int).Add(balancesBefore[i], big.NewInt(amount))) != 0 {
```